finesse/gui/data_file_view.py

```diff
@@ -0,0 +1,45 @@
+"""Provides a panel which lets the user start and stop recording of data files."""
+from pubsub import pub
+from PySide6.QtWidgets import QGroupBox, QHBoxLayout, QLabel, QPushButton
+
+from ..config import DEFAULT_DATA_FILE_PATH
+from .path_widget import SavePathWidget
+
+
+class DataFileControl(QGroupBox):
+    """A panel which lets the user start and stop recording of data files."""
+
+    def __init__(self) -> None:
+        """Create a new DataFileControl."""
+        super().__init__("Data file")
+
+        layout = QHBoxLayout()
+        layout.addWidget(QLabel("Path"))
```

Comment:
I know it's there on the old GUI, but this label seems a bit superfluous, especially considering there's not a corresponding label in the "Script control" panel.

Reply:
Good point